url_filter/filters.py

```diff
@@ -476,14 +477,27 @@ def __init__(self, form_field=None, *args, **kwargs):
         # need to overwrite to make form_field optional
         super(CallableFilter, self).__init__(form_field, *args, **kwargs)
 
+    @cached_property
+    def is_field(self):
+        """Return True if the attribute is instance of Field else return False.
+        """
+        model = self.root.queryset.model
```

Comment:
root queryset is the top-level queryset but since filtersets can be nested model relations need to be followed to get the child model to check if it is a field. at this time there is no generic way of getting the child model which gets more complicated with other backends